client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,67 @@
 import React from 'react';
+import axios from 'axios';
 
-const MovieCard = props => {
-  return;
-};
+
+class MovieCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movies: []
```

Comment:
If this is a MovieCard component that represents the display for a single movie card, why store an array on state?